CONTRIBUTING.md

```diff
@@ -140,3 +140,50 @@ To add a native module that wraps a FluentUI Apple control:
       - This Swift file imports FluentUI Apple, and creates a subclass of RCTViewManager to instantiate and return your FluentUI Apple control. Objective-C methods like `requiresMainQueueSetup` and `constantsToExport` can be overridden here. It's important to note that in order for properties and methods to be available to React Native, they must add the `@objc` decorator to it's declaration.
    1. `MSF<new-component>ViewManager.m`
       - This is an extra Objective-C file needed because Swift does not support macros and React Native requires them to map JS props to the native properties of the control. (Macros like `RCT_EXPORT_VIEW_PROPERTY` and `RCT_EXPORT_METHOD`)/
+
+## Creating a pull request
+
+Thanks for your interest in contributing to the fluentui-react-native! We welcome all contributions. Here's information on how to prepare your change for a pull request.
+
+### Code of Conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+### Prerequisites
+
+#### Beachball
+
+This repo manages semantic versioning and publishing using [Beachball](https://github.com/microsoft/beachball). When contributing, make sure to run the following before making a pull request:
+
+1. `yarn change` will take you through a command line wizard to generate change files
+2. Make sure to commit and push the newly generated change file
```

Comment:
nit: `yarn change` should auto commit the generated files for you, right?

Reply:
Yeah actually I think you're right. I'll remove that line.

Reply:
Fixed now!